packages/web/src/js/TextArea.ts

```diff
@@ -0,0 +1,47 @@
+import BaseComponent from './BaseComponent';
+import { enableToggleAutoloader } from './utils';
+import EventHandler from './dom/EventHandler';
+import { debounce } from './utils/Debounce';
+
+const NAME = 'textArea';
+const RESIZE_EVENT = 'resize';
+const DEBOUNCE_DELAY = 50;
+
+class TextArea extends BaseComponent {
+  textarea: HTMLTextAreaElement;
+
+  static get NAME() {
+    return NAME;
+  }
+
+  static get DATA_KEY() {
+    return `${this.NAME}`;
+  }
+
+  constructor(element: HTMLElement) {
+    super(element);
+    this.textarea = this.element.querySelector('textarea') as HTMLTextAreaElement;
+    this.adjustHeight = debounce(this.adjustHeight.bind(this), DEBOUNCE_DELAY);
```

Comment:
This is a little bit weird. You are assigning a denounced function based on this.adjustHeight method and then call it, I hope.

Reply:
Refactored, see now. Please.